api_tests/src/swap_response.ts

```diff
@@ -53,23 +53,23 @@ export interface Properties {
  */
 export enum SwapStatus {
     /**
-     * The swap was created but the communication phase hasn't yet been finalized
+     * The swap was created via the REST API but the communication phase hasn't yet been finalized.
```

Comment:
Sorry, I missed that a new state `CREATED` was added.

I discussed this with @thomaseizinger recently and was under the impression, that there should not be a new `CREATED` state in the aggregated swap state. `IN_PROGRESS` means that `cnd` was made aware of the swap.

All the other states are final states (`SWAPPED`, `NOT_SWAPPED`, `INTERNAL_FAILURE`).

`CREATED` for me is within the communication protocol.

What was the motivation for adding this state? 😬

Reply:
I don't think there is a point in discussing these states now :)
I would suggest to do whatever we think works currently. I see us adopting these names/states anyway to something that makes more sense once everything is in place.

Reply:
It is not new.

This was added as part of the design of the swap response on the cnd REST API many moons ago.

It is useful to test few things in the e2e test so I'd be keen to keep it.